From 59ef4e88f14d70d2870c6231df005d0ff98aabb4 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Wed, 1 Jul 2020 14:40:10 +0000
Subject: [PATCH] zynqmp-aes: Return negative error code on fail

---
 drivers/crypto/zynqmp-aes.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/crypto/zynqmp-aes.c b/drivers/crypto/zynqmp-aes.c
index 43e6610dd0a7..6face77727d4 100644
--- a/drivers/crypto/zynqmp-aes.c
+++ b/drivers/crypto/zynqmp-aes.c
@@ -197,6 +197,9 @@ static int zynqmp_aes_xcrypt(struct blkcipher_desc *desc,
 			dev_err(dd->dev, "ERROR: Invalid");
 			break;
 		}
+
+		err = -ret;
+
 		goto END;
 	}
 	if (flags)
-- 
2.17.1

